packages/blocks/src/blocks/CoreCode.tsx

```diff
@@ -60,3 +57,7 @@ CoreCode.fragments = {
     }
   `,
 };
+CoreCode.config = {
+  name: 'CoreCode',
+};
+CoreCode.displayName = 'CoreCode';
```

Comment:
Is there a reason why we are not putting both properties in the config?

```js
CoreCode.config = {
  name: 'CoreCode',
  CoreCode.displayName = 'CoreCode';
};
```

I assume this is so Webpack doesn't mangle the property name?